src/CoreShop/Bundle/StoreBundle/CoreExtension/StoreMultiselect.php

```diff
@@ -27,4 +27,9 @@ class StoreMultiselect extends Multiselect
      * @var string
      */
     public $fieldtype = 'coreShopStoreMultiselect';
+
+    public function getOptionsProviderClass()
+    {
+        return '@'.StoreOptionProvider::class;
```

Comment:
'@'?

Reply:
See https://github.com/pimcore/pimcore/blob/4550a053fac47a1d2c0406bf15c9799ef078bbb7/models/DataObject/ClassDefinition/Helper/ClassResolver.php#L35-L40
Because we need dependency injection here, we want to use the symfony service. And this is done with the prepending `@`.